xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -309,8 +309,11 @@
       #set ($filterType = 'text')
     #end
   #end
+  #set ($selectTitle = $escapetool.xml($services.localization.render('platform.livetable.filtersTitle',
```

Comment:
The variable name seems wrong as this is not just used for selects, maybe just `$filterTitle`?

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2149/commits/588a4e553cec58dfd235a1e66c4c0eea3a138c70